.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: "Unit Tests"
+
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    name: Running Unit Tests
+    runs-on: ubuntu-latest
+    container: cfcli/cli-base:latest
```

Comment:
I'm wondering if using a generic golang base image would future prof us by reducing one dependency.
I'm thinking about [this](https://github.com/cloudfoundry/cli/blob/5b57d4f0afdcdcda18adc1805d269c4e9164d3c7/.github/workflows/units.yml#L42-L45)

Reply:
In the past, cli-base was pretty much the only option, but with GHA we have more choices.